beacon_node/beacon_chain/src/light_client_finality_update_verification.rs

```diff
@@ -110,24 +94,13 @@ impl<T: BeaconChainTypes> VerifiedLightClientFinalityUpdate<T> {
             None => return Err(Error::SigSlotStartIsNone),
         }
 
-        let head_state = &head.snapshot.beacon_state;
-        let finality_update = LightClientFinalityUpdate::new(
-            &chain.spec,
-            head_state,
-            head_block,
-            &mut attested_state,
-            &finalized_block,
-        )?;
-
         // verify that the gossiped finality update is the same as the locally constructed one.
-        if finality_update != light_client_finality_update {
+        if latest_finality_update != rcv_finality_update {
             return Err(Error::InvalidLightClientFinalityUpdate);
```

Comment:
Not part of this PR, but this validation error is treated as a low tolerance error (peer gets downscored significantly) - does it make sense to just downscore slightly on this one?

https://github.com/sigp/lighthouse/blob/ba891e1fed8379f1410f50ce56eccc05f25daac2/beacon_node/network/src/network_beacon_processor/gossip_methods.rs#L1690-L1703

Reply:
Yes agree, clients can naturally disagree if one process the head update latter than another. It's not a very adversarial action